### ISSUE TITLE
Display goes to sleep/cannot wake mid process

### ISSUE THREAD

**Issue by PaddyMonahanLBNY** [rescuezilla/rescuezilla#369]
Hello, I am using the latest release of Rescuezilla (rescuezilla-2.4 64 bit jammy) through ventoy on a Lenovo Thinkpad 330 8th gen i3.
When I first boot I have to start in graphics fallback mode (didn't have this issue using it on my other devices: Toshiba Satellite c55-b, Asus eee-PC 1015pe) but everything seems to work fine. I initiated gparted to shrink the target disk and while it was doing this, an unknown amount of time had passed, the screen blacked out. I could hear the drives working so it was still functioning, but I couldn't wake the display. I held down the power button and hard shut down the laptop. Ran scans of the target file system(using standard ubuntu and then the actual target OS), made sure it could boot, then ran a checkdisk on it to be sure. 
Once the check is complete I run rescuezilla again in fallback mode. I begin backing up the target drive onto an external and it is running fine for some time, again indeterminate since I was working on other things, the screen blacks out. Again I can hear the drives writing, and I decide to leave this running for about the time I remember seeing it would take to write the image, but it never stopped or slowed or brought the screen back. I pressed all manner of keys and combinations to wake it (ctrl+alt+delete, alt+ctrl+1-6, ctrl+1-6, etc) but again the screen would not wake. I again had to hard shut down the device and again I am doing a disk scan. 

I had used clonezilla in the past and I have only just recently begun using rescuezilla but I have used both about equally as much so I can say this isn't something I've experienced outside of this device.

Any info or suggestions would be helpful, and I will provide whatever info is requested

**Comment by PaddyMonahanLBNY** [comment]
Also, I attached an external monitor VIA HDMI but not output/signal

**Comment by PaddyMonahanLBNY** [comment]
OK - i downloaded and attempted to use an older version (2.3.1.48) and it seems to have gotten a lot further than 2.4. What I believe to be the issue is that in graphical fallback mode, the error caused by trying to clone a drive with a bad sector seems to maybe have frozen or seized the display, as I didn't check the box to ignore bad sectors, and it happened roughly 25% into the process. When using 2.3.1.48 it slowed down to a dead stop around 25%, but I had checked to skip bad sectors so no error this time. I thought it could have been the issue since I attempted to use gparted from a live Linux Mint ISO and it gave me an error when shrinking the destination drive saying it couldn't be completed. Probably the same thing happened when using gparted on rescuezilla in fallback mode, the error message stating something related to bad sector reads/writes caused the visual output to shut down. AFAICT

**Comment by shasheene** [comment]
@PaddyMonahanLBNY  great to hear you've made progress. I'll have a deeper look at this ticket at a later point.

**Comment by PaddyMonahanLBNY** [comment]
BTW using 2.3.1.48 was completely successful, so far no problems with the cloned SSD drive, it seems to  have copied all the data cleanly and accurately despite bad sectors, and gave no issues with graphical interface or errors causing display failure.

**Comment by PaddyMonahanLBNY** [comment]
Good evening - experiencing same issue with new version of Rescuezilla, 2.4.1. The display goes to sleep mid-process and cannot be woken up, and I need to use graphic fallback mode again as well. I am now attempting to use Rescuezilla on an Acer Aspire E5-571 intel i3 1.90 Hz

**Comment by jmwilkes76** [comment]
i wonder if this is a lenovo problem. i'm also having the same issue with 2.4.1 on a lenovo m93p mini. i started to create a disk image and the screen blanked out. i wiggled my mouse it came back on. some times passed and the screen blanked out again, only this time it won't come back on.

**Comment by dr04e606** [comment]
It's definitely not just a Lenovo problem. I also have the same issue with display sleep on Acer TravelMate P259 in Rescuezilla, 2.4.1.

**Comment by shasheene** [comment]
Sorry to hear, but glad the Graphical Fallback Mode works.

Note: the v2.4.2 fix is only relevant to initial boot, not about sleep mode so it definitely won't fix your problem here.

**Comment by dr04e606** [comment]
> Note: the v2.4.2 fix is only relevant to initial boot, not about sleep mode so it definitely won't fix your problem here. 

Just checked the latest releases on this laptop: 

**v2.4.2-kinetic** boots to black screen if I don't select graphics fallback mode and still won't turn its screen back on after entering display sleep. 

**v2.4.2-bionic** works perfectly fine on this laptop.

**Comment by shasheene** [comment]
Interesting to know. Potentially because the Bionic variant doesn't have the drivers for power management for your hardware. Note: The bionic variant is only available as 32-bit on Rescuezilla v2.4.2. Using a 32-bit distribution on 64-bit hardware with more than a few gigabytes of RAM has very weird slowdown issues, so it's definitely not recommeded using it.

Also since Ubuntu 18.04 "Bionic" was released in April 2018, its security updates will soon drop (after 5 years) so I'll be dropping support for that variant in future releases.

**Comment by loucs93** [comment]
hi,

i had an almost similar issue on my desktop pc with the latest version (2.6.2). (ryzen 7 9800x3d, ASUS ROG STRIX X870E-E Gaming wifi, RTX 5070TI, and a 4k OLED monitor, plugged in the RTX GPU via HDMI 2.1). No graphical issue at first, no graphics fallback mode needed, just unreadable text due to the 4k resolution, had to go in 1080p. During the backup process, after some minutes of not using the mouse, the monitor blacked out so i tried using my mouse and keyboard to wake it up, but it stayed in sleep mode with no signal from HDMI. I tried to shut it down then powered it on again, but still no signal. sawing that my external drive used to store the backup image was still working, i waited and did something else. When i saw the drive wasn't active anymore, i tried moving my mouse and my screen woke up nicely, showing me the dialog saying that the backup was sucessfull. Not sure if i should have created a new issue as it's not perfectly similar, but didn't want to flood anyone. thanks to the contributors, and to those who read this post ^^.
